qt/scientific_interfaces/Inelastic/Manipulation/InelasticDataManipulationElwinTab.cpp

```diff
@@ -78,6 +57,7 @@ class ScopedFalse {
   ~ScopedFalse() { m_ref = m_oldValue; }
 };
 
+auto const regDigits = std::regex("\\d+");
```

Comment:
It might be possible to evaluate this at compile time, giving us a slight performance improvement at runtime
```suggestion
auto constexpr REG_DIGITS = std::regex("\\d+");
```

Reply:
I think is not possible with regexes. If you have any other solution?

Reply:
Ah ok, in that case ignore me!